tests/unit/modules/network/ios/test_ios_static_routes.py

```diff
@@ -2214,3 +2214,47 @@ def test_ios_static_route_gathered(self):
         self.maxDiff = None
         print(result["gathered"])
         self.assertEqual(sorted(result["gathered"]), sorted(gathered))
+
+    def test_ios_static_route_gathered_2(self):
```

Comment:
@roverflow let's add an action state test, just to be extra sure.
Anything works integration/ units!